project/Build.scala

```diff
@@ -251,8 +242,8 @@ object DottyBuild extends Build {
 
       // enable verbose exception messages for JUnit
       testOptions in Test += Tests.Argument(
-        TestFrameworks.JUnit, "-a", "-v",
-        "--run-listener=dotty.tools.ContextEscapeDetector"
+        TestFrameworks.JUnit, "-a", "-v"/*,
+        "--run-listener=dotty.tools.ContextEscapeDetector"*/
```

Comment:
Until we have a published new backend and https://github.com/lampepfl/dotty/pull/1894 is merged, yes. (I'll wait for these things to happen before merging this PR)